src/Administration/Resources/app/administration/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "administration",
-    "version": "1.0.0",
+    "version": "6.2.1",
```

Comment:
@granitibrahimi Please can you revert this change? - The version should be `1.0.0`